src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,40 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Storage {
+    private static Path storagePath;
+    private static File storageFile;
+
+    public static void initialise() {
+        try {
+            Path storageFolderPath = Paths.get("data");
+            if (Files.notExists(storageFolderPath)) Files.createDirectory(storageFolderPath);
+            Path storageFilePath = storageFolderPath.resolve(Paths.get("duke.tasks.txt"));
+            if (Files.notExists(storageFilePath)) Files.createFile(storageFilePath);
```

Comment:
I feel the loop body should be wrapped by curly brackets irrespective of how many lines there are in the body.

Reply:
Like what @jxt00 mentioned, do consider using curly brackets for the if statement, I think it will improve readability and adhere to the coding standards.
```suggestion
            if (Files.notExists(storageFolderPath)) {
                Files.createDirectory(storageFolderPath);
            }
            Path storageFilePath = storageFolderPath.resolve(Paths.get("duke.tasks.txt"));
            if (Files.notExists(storageFilePath)) {
                Files.createFile(storageFilePath);
            }
```

Reply:
+1 to the wrapping of the if-statement, I was also guilty of committing this, but it's part of the module's convention, and I think it does help with readability (even though it takes up a bit more space 😆 )